### PR TITLE
Changelog 1.20.4

### PR DESCRIPTION
**Pull request by lego963**

### CODE REVIEW

**Review by anton-sidelnikov — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
+* `resource/opentelekomcloud_compute_instance_v2`: Revert changes from [#686](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/686) ([#701](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/701))
+* `resource/opentelekomcloud_rds_instance_v3`: Fix reading nonexistent RDS instance ([#704](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/704))
+* `opentelekomcloud_provider`: Fix region handling ([#697](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/697))
+
+ENHANCEMENTS:
+* `resource/opentelekomcloud_cce_cluster_v3`: Add new fields `kubernetes_svc_ip_range` and `kube_proxy_mode` ([#699](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/699))
+* `resource/opentelekomcloud_lb_certificate_v2`: Add new fields `http2_enable`, `client_ca_tls_container_ref` and `tls_ciphers_policy` ([#688](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/688))
```
Comment (outcatcher) [inline CHANGELOG.md:15]: This PR relates to both `opentelekomcloud_lb_certificate_v2` and `opentelekomcloud_lb_listener_v2`
Alsso it fix retries for 409 and 503 errors

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,15 +1,29 @@
 ## 1.22.0 (Unreleased)
 
+
+## 1.21.4 (November 12, 2020)
+
+BUG FIXES:
+* `resource/opentelekomcloud_s3_bucket`: Fix panic creating `s3_bucket` without `tenant_name` in provider config ([#698](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/698))
+* `resource/opentelekomcloud_compute_instance_v2`: Revert changes from [#686](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/686) ([#701](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/701))
+* `resource/opentelekomcloud_rds_instance_v3`: Fix reading nonexistent RDS instance ([#704](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/704))
```
Comment (outcatcher) [inline CHANGELOG.md]: The main goal of this PR is fixing RDS instance import

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,15 +1,29 @@
...
+* `opentelekomcloud_provider`: Fix region handling ([#697](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/697))
```
Comment (outcatcher) [inline CHANGELOG.md]: Judging by number, this PR should be the first on the list

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,15 +1,29 @@
...
+
+ENHANCEMENTS:
```
Comment (outcatcher) [inline CHANGELOG.md:13]: Please order them by number asc

**Review by outcatcher — CHANGES_REQUESTED** [review]

**Review by anton-sidelnikov — DISMISSED** [review]

**Review by outcatcher — APPROVED** [review]

**Pull request merged**